test/fnull.js

```diff
@@ -0,0 +1,45 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('fnull', function() {
+  context('given function and default arguments', function() {
+    specify(
+      'should call the function with the given argument and default argument when one of the argument in null',
+      function() {
+        const add = (a, b) => a + b;
+
+        const safeAdd = RA.fnull(add, [1, 1]);
+
+        const value = safeAdd(1, null);
+
+        assert.equal(value, 2);
+      }
+    );
+
+    specify(
+      'should call the function with the given argument and default argument when one of the argument in undefined',
+      function() {
+        const add = (a, b) => a + b;
+
+        const safeAdd = RA.fnull(add, [1, 1]);
+
+        const value = safeAdd(1, undefined);
+
+        assert.equal(value, 2);
+      }
+    );
+  });
+
+  it('should curry', async function() {
+    const add = (a, b) => a + b;
```

Comment:
I suggest to simply it into following form. Also the function doesn't need to be async. In this form IMHO it is more apparent what we're testing here.
```js
it('should be curried', function() {
   const add = (a, b) => a + b;
   
   assert.strictEqual(RA.fnull(add, [1, 1]), 2);
   assert.strictEqual(RA.fnull(add)([1, 1]), 2);
});
```